cmd/youtubedr/output_format.go

```diff
@@ -37,10 +38,16 @@ func checkOutputFormat() error {
 
 type outputWriter func(w io.Writer)
 
-func writeOutput(w io.Writer, v interface{}, plainWriter outputWriter) error {
+func writeOutput(w io.Writer, v interface{}, writers map[string]outputWriter) error {
 	switch outputFormat {
+	case outputVideoIds:
+		fallthrough
 	case outputFormatPlain:
-		plainWriter(w)
+		writer, ok := writers[outputFormat]
+		if !ok {
+			return errUnhandledFormat(outputFormat)
```

Comment:
the flag is handled by `addFormatFlag` which prints 4 formats. For info cmd, this new format is not implemented. Maybe when info cmd accepts multiple URLs we can extract all video IDs the same way as list cmd does in this PR?

```
./youtubedr-dev info <videoID> --format video-ids
unhandled output format: video-ids
```

```
./youtubedr-dev list <playlistID> --format video-ids
JTRqppn4vNo lhbytXY9gEs 7Y7m3uMRNak GjhRMetsok0 fePN2EFK5bU 7sX44EkgTds sAb-ogxZIAI kv-8SeTfT0k CHvhcM_Rvlo -075jrjea4Q ZgHrrkG175M KK58lRTSn5I iIfgwENiL3o VNNXO4DIHUo
```